neofetch

```diff
@@ -2183,12 +2183,26 @@ END
         ;;
 
         "urxvt" | "urxvtd" | "rxvt-unicode" | "xterm")
-            term_font="$(grep -i "${term/d}\**\.*font" < <(xrdb -query))"
+            local _xrdb; _xrdb="$(xrdb -query)"
```

Comment:
No need for `local` or the `_var` naming convention. (*I know it's bad but Neofetch uses globals throughout and I want to keep consistency.*)

```sh
xrdb="$(xrdb -query)"
```